sycl/include/sycl/ext/intel/esimd/math.hpp

```diff
@@ -1851,6 +1851,39 @@ __ESIMD_API uint64_t rdtsc() {
 #endif
 }
 
+/// Performs clamping of values in a vector between min and max values.
+/// @tparam T type of the vectors.
+/// @tparam N size of the vectors.
+/// @param src vector containing an input.
+/// @param min vector containing minimum values.
+/// @param max vector containing maximum values.
+/// @return vector containing clamped input values between minimum and maximum.
+template <typename T, int N>
+__ESIMD_API __ESIMD_NS::simd<T, N> clamp(__ESIMD_NS::simd<T, N> src,
+                                         __ESIMD_NS::simd<T, N> min,
```

Comment:
I would not name the parameters `min` and `max` after having those issues with min and max macros on Windows defined in some system headers and disable-able by NOMINMAX.